generator/mavparse.py

```diff
@@ -290,11 +291,22 @@ def char_data(data):
                 self.include.append(data)
 
         f = open(filename, mode='rb')
+        from lxml import etree
+        xmldocument = etree.parse(f)
+
+        if not self.with_wip:
+            # remove wip elements from parsing
+            wip_entry = xmldocument.findall('.//wip/..')
```

Comment:
FYI This is a bit messy because you can't delete an object from itself. Here I find the wip parents, then delete each of them from their parent.